public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>canarias_dev</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons">
+  </head>
+  <body>
+    <noscript>
+      <strong>We're sorry but canarias_dev doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```

Comment:
😬

Reply:
Come on, we live in the 21st century, if it's a developer's page is understood not to connect from a toaster of the 90's

Reply:
![](https://media.giphy.com/media/d5Fbtoj47ROk0J9IgW/giphy.gif)

Reply:
🤣🤣🤣🤣🤣